src/bijectors/planar_layer.jl

```diff
@@ -121,16 +157,21 @@ which implies ``α̂ ∈ [wt_y - |wt_u_hat|, wt_y + |wt_u_hat|]``.
 D. Rezende, S. Mohamed (2015): Variational Inference with Normalizing Flows.
 arXiv:1505.05770
 """
-function find_alpha(wt_y, wt_u_hat, b)
-    # Compute the initial bracket.
+function find_alpha(wt_y::Real, wt_u_hat::Real, b::Real)
```

Comment:
A bit unrelated since never tested it seems (so maybe it's not useful?): one should define ChainRules rules for this function if one wants to differentiate through the inverse flow - even if it would be optimized by NonlinearSolve at some point, I assume we can always do better here. And everything is available in closed form.

Reply:
Maybe just add a "TODO" comment?